### ISSUE TITLE
Expand the introduction

### ISSUE THREAD

**Issue by zcorpan** [w3c/webvtt#230]
Should have more stuff in the introduction. Look at https://developer.mozilla.org/en-US/docs/Web/API/Web_Video_Text_Tracks_Format

Ack @r12a

**Comment by dwsinger** [comment]
I would prefer a simple patch that references, informatively, useful introduction(s) such as this.

**Comment by r12a** [comment]
My only worry with that is what happens if MDN disappears next month, or the URL changes.

**Comment by dwsinger** [comment]
Yes.  we had an introduction on webplatformdocs, but that site is no longer visible. it would be good to retrieve that content. https://web.archive.org/web/20160421023453/http://docs.webplatform.org/wiki/concepts/VTT_Captioning
